src/App.scss

```diff
@@ -2,7 +2,125 @@ iframe {
   display: none;
 }
 
+.title {
+  font-size: 50px;
+}
+.containerForApp {
+  width: 100%;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.App{
+  border: 2px black solid;
+  display: flex;
+  flex-direction: column;
+  width: 75%;
+  align-items: center;
```

Comment:
yagni: Consider using a more responsive approach for the `.App` element by using a max-width instead of fixed `75%` width. This will make the layout look better on different screen sizes. 🖥️